packages/workbox-strategies/src/Strategy.ts

```diff
@@ -168,11 +171,37 @@ abstract class Strategy implements RouteHandlerObject {
 
   async _getResponse(handler: StrategyHandler, request: Request, event: ExtendableEvent) {
     await handler.runCallbacks('handlerWillStart', {event, request});
-    let response = await this._handle(request, handler);
+
+    let response: Response | undefined = undefined;
+    try {
+      response = await this._handle(request, handler);
```

Comment:
What do you think about the case where `_handle()` responds with a `Response.error()` object?

Reply:
I hadn't thought about that

 Looks like it can be [detected](https://fetch.spec.whatwg.org/#responsetype) via `response.type === 'error'`. I'll add in support for triggering `handlerDidError` when that's encountered.

Reply:
Updated.